src/index.html

```diff
@@ -7,12 +7,69 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="image of IMAC"
+        class="card__picture-imac"
+      />
+
+      <div class="card__info">
+        <h1 class="card__name-product-imac">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
```

Comment:
This tag used for title of page, so will be better if you use the another tag here
```suggestion
        <h2 class="card__title">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>
```